applications/services.py

```diff
@@ -3,6 +3,10 @@
 from applications.models import Application, Event, Score
 from core.models import User
 
+def get_default(order):
```

Comment:
Could you please add a docstring that explans what this is for & what it is doing?

Reply:
Ya sure!